spec/src/main/asciidoc/core/definition.asciidoc

```diff
@@ -619,6 +636,8 @@ public class CoffeeShop
 Note that to ensure compatibility with other Jakarta Dependency Injection implementations, all pseudo-scope annotations except `@Dependent` *are not* bean defining annotations.
 However, a stereotype annotation including a pseudo-scope annotation *is* a bean defining annotation.
 
+// TODO we might be able to make `@Singleton` a bean defining annotation in Lite? depending on how we decide on the beans.xml thing
```

Comment:
I don't think you can do that. Because in Full it cannot be bean defining annotation and that would create inconsistency between Lite and Full.

Reply:
Fair point.